src/qibolab/qubits.py

```diff
@@ -15,7 +15,15 @@
 
 Not all channels are required to operate a qubit.
 """
-EXCLUDED_FIELDS = CHANNEL_NAMES + ("name", "native_gates", "kernel", "_flux")
+EXCLUDED_FIELDS = CHANNEL_NAMES + (
+    "name",
+    "native_gates",
+    "kernel",
+    "_flux",
+    "qubit1",
+    "qubit2",
```

Comment:
https://github.com/qiboteam/qibolab/blob/00ade29d9c9e88b6c822a28a73fdae3590aeb637/src/qibolab/qubits.py#L166
since now `EXCLUDE_FIELDS` is also used in `QubitPair`
https://github.com/qiboteam/qibolab/blob/00ade29d9c9e88b6c822a28a73fdae3590aeb637/src/qibolab/qubits.py#L193

Reply:
I now wonder whether it would be a better option to have one each...

Btw, we don't even strictly need the `EXCLUDED_FIELDS` constant, but we could use `field(..., metadata={"excluded": True})` or use `Annotated[..., EXCLUDED]` as type hint (defining an `EXCLUDED` constant), to keep it local to the ignored field.

Reply:
(a possible way of defining unique constants can be found in the standard library https://github.com/python/cpython/blob/b03d71d0d8415281e71ae4f1455222db4b4b66d2/Lib/dataclasses.py#L184-L186)